<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeInfo</class>
 <widget class="QDialog" name="EmployeeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="personal">
        <attribute name="title">
         <string>Personal Information</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="horizontalSpacing">
            <number>22</number>
           </property>
           <property name="verticalSpacing">
            <number>32</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Machine Id</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leMachineId"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Phone</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lePhone"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>First Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leFirstName"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Another Email</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="leAnotherEmail"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Last Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leLastName"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Mobile</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="leMobile"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="leEmail"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Hour Price</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="leHourPrice"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="leaves">
        <attribute name="title">
         <string>Leaves</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTableView" name="tvLeaves"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="pbtnAddLeave">
               <property name="text">
                <string>Add Leave</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtnDelLeave">
               <property name="text">
                <string>Delete Leave</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>108</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="retribution">
        <attribute name="title">
         <string>Retributions</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableView" name="tvRetribution"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pbtnAddRetribution">
               <property name="text">
                <string>Add Retribution</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtnDelRetribution">
               <property name="text">
                <string>Delete Retribution</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="bounces">
        <attribute name="title">
         <string>Bounces</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableView" name="tvBounce"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="pbtnAddBounce">
               <property name="text">
                <string>Add Bounce</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtnDeleteBounce">
               <property name="text">
                <string>Delete Bounce</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnUpdate">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbtnClose</sender>
   <signal>clicked()</signal>
   <receiver>EmployeeInfo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
